src/components/ContactUsForm/ContactUs.jsx

```diff
@@ -33,38 +94,226 @@ const ContactUs = () => {
       been_dismissed: false,
     };
     const result = await backend.post('/notification/', updatedFormData);
+    console.log(result);
+    setShowConfirmation(true);
+  };
+
+  const handleNumInputChange = (e, index) => {
+    checkedItems[index] = Number(e);
+
+    // Update the state with the new array
+    setCheckedItems(checkedItems);
+    isFormFilled();
+    setIsFormValid(checkedItems.some(value => value !== 0) && text.length > 0);
+    console.log(checkedItems);
   };
 
-  const handleInputChange = e => {
-    let inputValue = e.target.value;
-    setText(inputValue);
+  const isFormFilled = () => {
+    // Check if all form fields are filled
+    console.log(checkedItems.some(value => value !== 0) && text.length > 0);
+    return checkedItems.some(value => value !== 0) && text.length > 0;
   };
 
   const setMessage = text => {
     const textCopy = text;
-    const checkedThingies = ['Stickers', 'Posters', 'Business Cards', 'Something', 'Joshua Lipton'];
-    const preMessage = `${name} is requesting: \n`; // business name... hard-coded right now
+    const checkedThingies = [
+      'DS Decals',
+      'Rack Cards',
+      'Donation Envelopes',
+      'Business Cards',
+      'Stickers',
+      'Bin Decals',
+      'DS Decals',
+    ];
+    const preMessage = `Business is requesting: \n`; // business name... hard-coded right now
     const updatedMessage = checkedThingies.filter((_, index) => checkedItems[index]).join('\n-');
     return `${preMessage}-${updatedMessage}\nNotes:\n${textCopy}`;
   };
 
+  const textInputHandler = e => {
+    const newText = e.target.value;
+    setText(newText);
+    isFormFilled();
+    setIsFormValid(checkedItems.some(value => value !== 0) && text.length > 0);
```

Comment:
make this an or so that businesses can request supplies without a message or send a message without requesting supplies